### DISCUSSION TITLE
SSB demodulation / Modulation

### DISCUSSION THREAD

**Discussion by CEHansen** [pschatzmann/arduino-audio-tools#879] [Q&A] [answered]
Will the tools package be able to do SSB or FM demodulation from a stream or AD-converter (IQ) input ?
Anyone trying this ?
/Claus

**Comment by pschatzmann** [comment, marked as answer]
I think you need to look for a proper signal processing library like liquid-dsp to do this.

**Reply by CEHansen** [reply]
Thanks for the Hint, I will have a look.
